lib/cassette.rb

```diff
@@ -38,9 +38,9 @@ def logger
   def config
     @config if defined?(@config)
 
-    @config.tls_version = DEFAULT_TLS_VERSION if @config.tls_version.nil?
+    @config.tls_version = DEFAULT_TLS_VERSION if @config&.tls_version.nil?
```

Comment:
@Wynteres  I think this safe navigator could be replaced for double pipes equal, don't it?

Reply:
Yes, in fact there was a miss conception here by my side, instead of using a safe navigator we must instantiate a new OpenStruct object if there is no `@config` defined. This will stop the error of happening for good